Source/Core/Resource.js

```diff
@@ -1755,26 +1825,50 @@ define([
      */
     Resource._Implementations = {};
 
-    Resource._Implementations.createImage = function(url, crossOrigin, deferred) {
-        var image = new Image();
+    Resource._Implementations.createImage = function(url, crossOrigin, deferred, flipY) {
+        if (!FeatureDetection.supportsCreateImageBitmap()) {
+            var image = new Image();
 
-        image.onload = function() {
-            deferred.resolve(image);
-        };
+            image.onload = function() {
+                deferred.resolve(image);
+            };
 
-        image.onerror = function(e) {
-            deferred.reject(e);
-        };
+            image.onerror = function(e) {
+                deferred.reject(e);
+            };
 
-        if (crossOrigin) {
-            if (TrustedServers.contains(url)) {
-                image.crossOrigin = 'use-credentials';
-            } else {
-                image.crossOrigin = '';
+            if (crossOrigin) {
+                if (TrustedServers.contains(url)) {
+                    image.crossOrigin = 'use-credentials';
+                } else {
+                    image.crossOrigin = '';
+                }
             }
+
+            image.src = url;
+        } else {
+            // createImageBitmap will block the main thread during decode if a blob is not supplied.
```

Comment:
Is this true for all browsers?  Do you have a citation that states this to be the case?  Please add the url to the comment so future readers can learn more.

Reply:
@OmarShehata did I miss an answer to this?